optuna/storages/rdb/models.py

```diff
@@ -11,6 +11,7 @@
 from sqlalchemy import orm
 from sqlalchemy import String
 from sqlalchemy import UniqueConstraint
+from sqlalchemy import desc, asc
```

Comment:
Could you import the two symbols in separate lines, for consistency with the above imports?

Reply:
done

Reply:
done )